applications/nrf5340_audio/README.rst

```diff
@@ -62,11 +62,6 @@ Regardless of the configuration, the application handles the audio data in the f
 
 #. Decoded audio data is sent to the hardware audio output over I2S.
 
-.. note::
```

Comment:
The first paragraph in [Overview](https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8915/nrf/applications/nrf5340_audio/README.html#id12) should be updated

Reply:
Done